WcaOnRails/app/controllers/competitions_controller.rb

```diff
@@ -278,6 +278,20 @@ def update_events
     end
   end
 
+  def update_wcif
+    @competition = competition_from_params
+    wcif = params.permit!.to_h
```

Comment:
Sure, or maybe `wcif_doc`? Is the ambiguity here whether this is a string, or a ruby object? I have this problem when dealing with JSON... I never know if "foo_json" is a string that can be intepreted as json, or if it is a parsed json object.